src/Contracts/Models/DicomTags.cs

```diff
@@ -1,17 +1,20 @@
-﻿using System.Collections.Generic;
-using Newtonsoft.Json;
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
 
 namespace Monai.Deploy.WorkflowManager.Contracts.Models
 {
     public class DicomTags
     {
-        [JsonProperty(PropertyName = "study_id")]
+        [ConfigurationKeyName("study_id")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
changed